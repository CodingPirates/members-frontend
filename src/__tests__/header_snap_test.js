import React from "react";
import { create } from "react-test-renderer";
import Header from "../components/header.js";
describe("Header Test", () => {
  test("it matches the snapshot", () => {
    const component = create(<Header />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
