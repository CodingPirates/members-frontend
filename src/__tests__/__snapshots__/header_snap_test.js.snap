// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test it matches the snapshot 1`] = `
<div
  className="sc-bdVaJa zZxdL"
>
  <div
    className="sc-bwzfXH dSjrNB"
  />
  <h5
    className="sc-htpNat hqQPgY"
  >
    Medlemssystem
  </h5>
  <img
    alt="Logo"
    className="sc-bxivhb leKRDg"
    src="logo.png"
  />
</div>
`;
